### DISCUSSION TITLE
v.4.0.1 - where are the indicators like "Ichimoku"?

### DISCUSSION THREAD

**Discussion by UweR70** [tradingview/lightweight-charts#1358] [General]
Did I miss something?

I am using **v.4.0.1** and I am able to setup an Angular app, include and use lightweight-charts.
For example this works fine:

`const chart: IChartApi = createChart(container, this.chartOptions);`
 
But where are the indicators? 
Specially Ichimoku?
 
`const ichimokuCloudIndicator = chart.addTechnicalIndicator('ichimoku', {
      conversionPeriod: 9,
      basePeriod: 26,
      spanPeriod: 52,
      displacement: 26,
    });`
 
 Shows
 TS2339 Property 'addTechnicalIndicator' does not exist on type "IChartApi".
 
 
Are the indicators removed 
(to sell another version/product)?

**Comment by UweR70** [comment]
So I wrote my own implementation ...
Thanks for your feedback ... oh yes ... I did not get any

**Comment by romfrancois** [comment]
Hi @UweR70 !

Apologies for missing this thread. There's a different level of notifications between issues & discussions and with all the Bank Holidays observed in the UK we have a backlog of tickets to go back to!

As for your question, Lightweight Charts doesn't come with indicators built in. You have to do it on your own as shown on our documentation or product page.

**Comment by UweR70** [comment]
I have to say sorry! 
And thanks for your answer.

In the meantime I´ve implememted my own Ichimoku indiactor - including the part that shows the Senkou A and B lines in future time periods; based on the given config values.

But now I see another topic on which I will ask a new question.
Again:
Sorry! My fault!
